example/lib/main.dart

```diff
@@ -42,6 +42,12 @@ class Custom extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return PagedVerticalCalendar(
+      weekdaysToHide: [
+        DateTime.saturday,
+        DateTime.sunday,
+      ],
+      startWeekWithSunday: true,
```

Comment:
no need to modify the example to include these customizations, since it's quite a specialized feature